libs/langchain/langchain/retrievers/dingo.py

```diff
@@ -0,0 +1,60 @@
+"""Dingo Retriever"""
+from typing import Any, Dict, List, Optional
+
+from pydantic import Extra, root_validator
+
+from langchain.callbacks.manager import CallbackManagerForRetrieverRun
+from langchain.embeddings.base import Embeddings
+from langchain.schema import BaseRetriever, Document
+from langchain.vectorstores.dingo import Dingo
+
+
+class DingoRetriever(BaseRetriever):
+    """Retriever that uses the Dingo API"""
+
+    embeddings: Embeddings
+    """Embeddings model to use."""
+    """description"""
```

Comment:
stray comment

Reply:
> stray comment
I hava Fixed， removed this """description"""